apps/cms/src/customize/graphql/query/test.ts

```diff
@@ -0,0 +1,30 @@
+export const testsCustomizeHandler = (tests) => {
+  tests.nodes.forEach((test) => {
+    const { createdBy } = test;
+
+    test.author = {
+      id: createdBy.id,
+      email: createdBy.email,
+      firstName: createdBy.firstname,
+      lastName: createdBy.lastname,
+    };
+  });
+
+  return tests;
+};
+
+export const testsCustomResolver = async (parent, args, context) => {
+  const { toEntityResponseCollection } = strapi
+    .plugin('graphql')
+    .service('format').returnTypes;
+
+  const data = await strapi.services['api::test.test'].find({
```

Comment:
Do we don't need anything here to filter the test?

Reply:
Thanks bro .Checked and updated